src/r3/atomic-swap/corda/evm-interop-workflows/src/main/kotlin/com/r3/corda/evminterop/services/swap/DraftTxService.kt

```diff
@@ -19,6 +19,7 @@ class DraftTxService(private val serviceHub: AppServiceHub) : SingletonSerialize
 
     private val transactions = ConcurrentHashMap<SecureHash, WireTransaction>()
     private val signatures = ConcurrentHashMap<BigInteger, HashSet<DigitalSignature.WithKey>>()
+    private val evmSignatures = ConcurrentHashMap<String, HashSet<ByteArray>>()
```

Comment:
Not really a functional comment but is there a reason this is a `String` and not a `SecureHash`?

Reply:
I'm currently reviewing the whole code changed in the last pull requests and there will be some refactoring, cleanups, improvements, etc. This was a temporary solution and will certainly be improved in the next PR.